tutorials/monitoring/setup/index.md

```diff
@@ -0,0 +1,209 @@
+
+
+### Install ServiceControl
+
+ServiceControl is a suite of back-end tools that collect useful information about your running system. The main installer includes a desktop utility called the ServiceControl Management Utility which can be used to create and manage ServiceControl and Monitoring instances.
+
+Download and run the latest [ServiceControl installer](https://github.com/Particular/ServiceControl/releases/download/1.42.0/Particular.ServiceControl-1.42.0.exe).
+
+NOTE: You should run the latest version of the installer, even if you already have ServiceControl installed. Only versions 1.43 and above of the ServiceControl Management Utility can create and manage Monitoring instances.
+
+Once it has finished installing, run the ServiceControl Management Utility.
+
+
+### Create ServiceControl instance
+
+NOTE: You can use an existing ServiceControl instance if it is version 1.43 or above.
+
+A ServiceControl instance collects messages from the error queue and manages retrying them. It also collects messages from the audit queue. 
+
+In the ServiceControl Management Utility, click **+ New** and select **ServiceControl instance** from the list of options.
+
+![ServiceControl Management Utility - Add new ServiceControl instance](scmu-add-new-sc-instance.png)
+
+Scroll down to the **Transport Configuration** section. Select the transport that your NServiceBus system runs on. Depending on the transport you may be required to add an additional connection string.
+
+Under **Audit Forwarding** select either On or Off. If you are not currently using an audit queue for anything, you can safely turn this Off. See [audit fowarding](/servicecontrol/errorlog-auditlog-behavior.md) for more information.
+
+Click the Add button. Your ServiceControl instance will start and be listed on the main page of the ServiceControl Management Utility.
+
+![ServiceControl Management Utility - List instance - ServiceControl instance](scmu-list-sc-instance.png)
+
+The listing for the ServiceControl instance includes a URL link. This URL will be needed when installing ServicePulse below.
+
+NOTE: Creating the ServiceControl instance will also create the audit and error queues if they did not already exist. By default these are called _audit_ and _error_ respectively.
+
+
+### Create Monitoring Instance
+
+A Monitoring instance collects data from the monitoring queue and aggregates information from all of the endpoints in the system.
+
+In the ServiceControl Management Utility, click **+ New** and select **Monitoring instance** from the list of options.
+
+![ServiceControl Management Utility - Add new Monitoring instance](scmu-add-new-mon-instance.png)
+
+Scroll down to the **Transport Configuration** section. Select the transport that your NServiceBus system runs on. Depending on the transport selected you may be required to add an additional connection string.
```

Comment:
Should we mention that SC and Monitoring should run on the same transport? I think only some funky setups could use different transports for those two.